lib/tasks/coursemology/migrate_comments.rake

```diff
@@ -0,0 +1,95 @@
+namespace :db do
+  task migrate_comments: :environment do
+    ActsAsTenant.without_tenant do
+      ActiveRecord::Base.transaction do
```

Comment:
I am not sure if the transaction will have conflicts with other user inserts, since the transaction will be not be committed for hours.